pyServer/AzureDiskInspectService.py

```diff
@@ -458,7 +469,14 @@ def do_POST(self):
             self.telemetryLogger.info('<<STATS>> ' + self.serviceMetrics.getMetrics())
 
             sasKeyStr = str(postvars[b'saskey'][0], encoding='UTF-8')
-            operationId, mode, modeMajorSkipTo, modeMinorSkipTo, storageAcctName, container_blob_name, storageUrl = self.ParseUrlArguments(self.path, sasKeyStr)                
+            if b'cloudenv' in postvars:
```

Comment:
LGTM. I suggest adding a quick UT to make sure this new parameter is parsed correctly.

Reply:
Created Work item for UT. Will be one more PR for UT but need this too be merge asap.
TASK 6438445